server/views/helpers/helpers.ejs

```diff
@@ -213,9 +208,9 @@ getDateTimeNowUTC = () => {
   return new Date(utc_timestamp);
 }
 
-getRelativeLastModifiedDateTime = (dateTimeInSeconds) => {
+getRelativeLastModifiedDateTime = (dateTime) => {
```

Comment:
Is this function tested? It would be nice to have assurance it still works the same